src/components/shared/__tests__/InfoTooltip.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { InfoTooltip } from '@/components/shared/InfoTooltip'
+
+describe("Checks that InfoTooltip snapshots don't change", () => {
+  it('renders correctly', () => {
+    const inlineClipboard = render(<InfoTooltip label={'label'} />)
```

Comment:
Should be `infoTooltip`. @Carminepo2 , we should add a test when hovered (label displayed)

Reply:
fixed name and  added hovered state snapshot